app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
- [ ] It would be nice if you connect your User with the below tables because when you make the one-to-many  association between the tables then the association consists of two things `has_many` and `belongs_to` :+1: Kindly add the has_many part in the user model to follow the requirements :+1:

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'user_id'
has_many :likes, foreign_key: 'user_id'
```